back/sssscs/src/main/java/com/ib/pki/data/IssuerData.java

```diff
@@ -0,0 +1,19 @@
+package com.ib.pki.data;
+
+import java.security.PrivateKey;
+
+import org.bouncycastle.asn1.x500.X500Name;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class IssuerData {
+	private PrivateKey privateKey;
```

Comment:
This class is also never used?

Reply:
Fixed in [b111018](https://github.com/magley/sssscs/pull/5/commits/b111018d31a0b6a52e6ee80782ef789c34c47a82).